### PR TITLE
DMS-1194 Change spec to include Flu and HPV

### PR DESCRIPTION
**Pull request by nathanwall**
## Summary
* Routine Change

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [x] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

**Comment by ProductBrett** [comment]
@nathanwall I'm not setup to review, I think we want the review by the Tech Author and not Product Owner

**Review by ProductBrett — APPROVED** [review]
Approved as Product Owner having previously reviewed with technical author

**Review by ProductBrett — APPROVED** [review]

File: specification/immunisation-history.yaml [hunk 1]
```diff
@@ -14,23 +14,25 @@ info:
...
+
     - immunisation event details
     - patient demographic details, as captured at the point of immunisation
 
-    The patient demographic details might differ from those held in the [Personal Demographics Service (PDS)](https://digital.nhs.uk/services/demographics).
-    To get demographic details from PDS, use the [Personal Demographics Service FHIR API](https://digital.nhs.uk/developer/api-catalogue/personal-demographics-service-fhir).
-    
-    If you are using this API in [user-restricted access mode](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation#user-restricted-apis) the patient must have their identity verified to 'high' (P9) level.
-    
+    The patient demographic details might differ from those held in the Personal Demographics Service (PDS). To get demographic details from PDS, use the Personal Demographics Service FHIR API.
```
Comment (ProductBrett) [inline specification/immunisation-history.yaml:34]: @nathanwall please update` Personal Demographics Service FHIR API` to
`[Personal Demographics FHIR API](https://digital.nhs.uk/developer/api-catalogue/personal-demographics-service-fhir)`

File: specification/immunisation-history.yaml [hunk 2]
```diff
@@ -40,6 +42,12 @@ info:
 
     In a very small number of cases, we are unable to verify the NHS number, and we do not make the immunisation record available at all.
 
+    The `COVID-19` data store is limited to vaccinations from England, Wales and The Isle of Man for the last 4 years. It has been approved to be used for clinical use cases.
+
+    The `FLU` data store is not yet populate and is planned to contain vaccination events captured through a point of care system in England for the last year. It has been approved to be used for clinical use cases.
```
Comment (ProductBrett) [inline specification/immunisation-history.yaml:47]: @nathanwall please update `populate` to `populated`

File: specification/immunisation-history.yaml [hunk 3]
```diff
@@ -40,6 +42,12 @@ info:
...
+
+    The `HPV` data store is not yet populated and is planned to be limited at first  to vaccinations captured by the NHAIS system and in England only. It has been approved for use in clinical use cases.
```
Comment (ProductBrett) [inline specification/immunisation-history.yaml:49]: Please replace '  ' with ' ' after the word `first`

**Review by ProductBrett — CHANGES_REQUESTED** [review]

**Review by nigelg — APPROVED** [review]

**Pull request merged**